bundles/org.openhab.ui.dashboard/templates/setup.html

```diff
@@ -22,6 +25,12 @@
         <h1>Welcome to openHAB 2 - Initial Setup</h1>
         <div class="line">
             <div class="decorator"></div>
+            <hr />
+            <div class="geolocation">
+                <p>In order to setup the location of your openHAB instance please accept the request from your browser. The location will be used for weather and date/time information.</p>
+                <p><span>Geolocation: </span><span id="geolocation"></span></p>
+            </div>
+            <div class="decorator"></div>
```

Comment:
If the location is already set and thus the div above is not shown, we end up with two decorator lines - can this be hidden as well in that case?